CODE_OF_CONDUCT.md

```diff
@@ -31,33 +31,27 @@ The following behaviors are expected and requested of all community members:
+ * Community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
 
-## 5. Weapons Policy
-
-No weapons will be allowed at any code curious events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon. Community members are further expected to comply with all state and local laws on this matter.
+#### Harassment includes:
+* harmful or prejudicial verbal or written comments related to gender, sexual orientation, race, religion, disability;
+* inappropriate use of nudity and/or sexual images (including presentation slides);
+* inappropriate depictions of violence (including presentation slides);
+* deliberate intimidation, stalking or following;
+* harassing photography or recording;
+* sustained disruption of talks or other events;
+* inappropriate physical contact, and unwelcome sexual attention.
 
 ## 6. Consequences of Unacceptable Behavior
 
-Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
-
-Anyone asked to stop unacceptable behavior is expected to comply immediately.
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated. Anyone asked to stop unacceptable behavior is expected to comply immediately.
 
 If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
 
 ## 7. Reporting Guidelines
 
-If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Email us at contact@codecurious.org, tweet us (@codecurius-bln). If you're having problems with organisers, you can contact us personally or contact Ferdous [under ferdous@codecurious.org or on twitter @ferbsx] or Yonatan [under yonatan@shuhugah.com or @shushugah on Twitter].
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Email us at contact@codecurious.org, tweet us (@codecurius-bln). If you'd rather speak directly to certain organisers, you can contact Ferdous in English or German [under ferdous@codecurious.org or on twitter @ferbsx] or Yonatan in English [under yonatan@shuhugah.com or @shushugah on Twitter] or Kaja in German or English [under kaja@codecurious.org or @AlizeNero].
```

Comment:
If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. 
### Contact information
- Email us at contact@codecurious.org
- Via Twitter direct messages [@codecurious_bln](https://twitter.com/messages/28624302-553622948).
 
If you would prefer to speak directly with individual organizers, you can contact these people
- Ferdous (English/Deutsch) by email ferdous@codecurious.org and Twitter [@ferbsx](https://twitter.com/@ferbsx) 
- Yonatan (English) by email yonatan@shuhugah.com and Twitter [@shushugah](https://twitter.com/@shushugah)
- Kaja (Deutsch/English) by email kaja@codecurious.org and Twitter [@AlizeNero](https://twitter.com/@AlizeNero)

Reply:
:+1: